plugins/api-docs/src/components/ApiExplorerPage/ApiExplorerPage.tsx

```diff
@@ -78,16 +77,10 @@ export const ApiExplorerPage = ({
     >
       <Content>
         <ContentHeader title="">
-          {createComponentLink && (
-            <Button
-              variant="contained"
-              color="primary"
-              component={RouterLink}
-              to={createComponentLink()}
-            >
-              Register Existing API
-            </Button>
-          )}
+          <CreateButton
+            title="Register Existing API"
+            to={createComponentLink && createComponentLink()}
```

Comment:
```suggestion
            to={createComponentLink?.()}
```

Could use optional chaining here if you like 🤷‍♂️